apps/keyboard/js/layouts/emoji.js

```diff
@@ -112,28 +157,34 @@ Keyboards.emoji = {
         {
           value: '',
           keyCode: KeyboardEvent.DOM_VK_ALT,
-          className: 'emoji-people active',
+          className: 'emoji-recent',
           targetPage: 0
         },
+        {
+          value: '',
+          keyCode: KeyboardEvent.DOM_VK_ALT,
+          className: 'emoji-people active',
```

Comment:
active is a style for selected key icon, which change it to another png.
This should not be removed to show user current tab.